src/frontend/Sider/Personoversikt/Vedtaksperioderoversikt/VedtaksperioderOversikt.tsx

```diff
@@ -30,35 +36,68 @@ export function VedtaksperioderOversikt({ fagsakPersonId }: Props) {
+                                        vedtaksperioder={vedtaksperioderOversikt.tilsynBarn}
+                                    />
+                                    <Detail>
+                                        Oppdatert: {formaterDatoMedTidspunkt(hentetTidspunkt)}
+                                    </Detail>
+                                </OversiktKort>
+                            )}
+                            {vedtaksperioderOversikt.læremidler.length > 0 && (
+                                <OversiktKort tittel={'Læremidler'}>
+                                    <VedtaksperioderOversiktLæremidler
+                                        vedtaksperioder={vedtaksperioderOversikt.læremidler}
+                                    />
+                                    <Detail>
+                                        Oppdatert: {formaterDatoMedTidspunkt(hentetTidspunkt)}
+                                    </Detail>
+                                </OversiktKort>
+                            )}
+                            {vedtaksperioderOversikt.boutgifter.length > 0 && (
+                                <OversiktKort tittel={'Boutgifter'}>
+                                    <VedtaksperioderOversiktBoutgifter
+                                        vedtaksperioder={vedtaksperioderOversikt.boutgifter}
+                                    />
+                                    <Detail>
+                                        Oppdatert: {formaterDatoMedTidspunkt(hentetTidspunkt)}
+                                    </Detail>
+                                </OversiktKort>
+                            )}
+                            {arenaSakOgVedtak.vedtak.length > 0 && (
+                                <OversiktKort
+                                    tittel={'TS vedtak i Arena'}
+                                    tagTittel={'Arena'}
+                                    tagVariant={'info'}
+                                >
+                                    <VedtaksperioderOversiktArena
+                                        arenaSakOgVedtak={arenaSakOgVedtak}
+                                    />
+                                    <Detail>
+                                        Oppdatert: {formaterDatoMedTidspunkt(hentetTidspunktArena)}
+                                    </Detail>
+                                </OversiktKort>
+                            )}
+                        </VStack>
+                    )
+                }
             </DataViewer>
         </>
     );
 }
+
+const finnesIngenVedtaksperioder = (
+    vedtaksperioderOversikt: VedtakperioderOversiktResponse,
+    arenaSakOgVedtak: ArenaSakOgVedtak
+): boolean =>
+    vedtaksperioderOversikt.boutgifter.length === 0 &&
+    vedtaksperioderOversikt.læremidler.length === 0 &&
+    vedtaksperioderOversikt.tilsynBarn.length === 0 &&
+    arenaSakOgVedtak.vedtak.length === 0;
```

Comment:
Gjerne putt denne i en util-fil hvis det finnes en som er relevant å bruke. Ikke nødvendig å lage en bare for denne